src/cmd/manage_nsfs.js

```diff
@@ -102,10 +102,13 @@ async function main(argv = minimist(process.argv.slice(2))) {
     } catch (err) {
         dbg.log1('NSFS Manage command: exit on error', err.stack || err);
         const manage_err = ((err instanceof ManageCLIError) && err) ||
-            new ManageCLIError(ManageCLIError.FS_ERRORS_TO_MANAGE[err.code] ||
+            new ManageCLIError({
+                ...(ManageCLIError.FS_ERRORS_TO_MANAGE[err.code] ||
                 ManageCLIError.RPC_ERROR_TO_MANAGE[err.rpc_code] ||
-                ManageCLIError.InternalError);
-        throw_cli_error(manage_err, err.stack || err);
+                ManageCLIError.InternalError), detail: err.stack || err });
```

Comment:
i think the detail here is not needed because it repeats the message

Reply:
oh you want it for the stacktrace? in that case, perhaps just add the stack trace to the to_string() function itself - it can get the stack from this.stack and then it can go into a stack field and not details...

Reply:
yes 👍🏻 I added it to the to_string()